src/state.rs

```diff
@@ -42,6 +42,7 @@ pub static NAMESPACE_ORDER_ASK_V2: &[u8] = b"ask_v2";
 pub static NAMESPACE_ORDER_BID_V2: &[u8] = b"bid_v2";
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
```

Comment:
The json looks weird without this rename_all.  Also it might be good to try to do some serde flattening for these because the output is like: ```{"base": {"scope": {"scope_address": ""}}}` which isn't great.  Stupid enums